noise/conn.go

```diff
@@ -17,6 +17,7 @@ type Conn struct {
 	// handshake
 	config            *Config // configuration passed to constructor
 	hs                handshakeState
+	remotePub         []byte
```

Comment:
why not making it a `[32]byte` to avoid having to `make` during the handshake?

Reply:
Becase I wanted to see if you can see it ? ahah. well spotted.